### DISCUSSION TITLE
Ender 3 w/ SKR Mini E3 V1.2

### DISCUSSION THREAD

**Discussion by rockman44181000** [Jyers/Marlin#1207] [General Discussion]
I've been using manual mesh bed leveling for quite some time now and really like the features and screens that have been implemented in this firmware. Will this work with the Ender 3 with the SKR Mini E3 v1.2 control board or is it only for the Ender 3 V2? My current setup is using the 7 x 7 grid with the G28 restore feature enabled along with the G48 command enabled as well and I am using the G420 S1 added to my start up configuration. This would certainly reduce the amount of time required to reflash my firmware and trying to remember all of the settings required doing so. Considering it has been quite some time since my last update.

**Comment by Cal-LF** [comment]
Jyres FW supports the E3V2 and it's screen. Users are finding ways to modify and compile for your mainboard. The screen is the deciding factor. It does not work with the Ender3 screen. I believe you could change to the V2 screen and get it working. Reference other recent posts about the SKR Mini E3.
